getting-started/kubernetes/vpp/getting-started.md

```diff
@@ -84,7 +84,7 @@ Before you get started, make sure you have downloaded and configured the {% incl
 1. Now that you have a cluster configured, you can install {{site.prodname}}.
 
    ```bash
-   kubectl apply -f https://raw.githubusercontent.com/projectcalico/vpp-dataplane/master/yaml/generated/calico-vpp-eks.yaml
+   kubectl apply -f https://raw.githubusercontent.com/projectcalico/vpp-dataplane/v0.16.0-calicov3.20.0/yaml/generated/calico-vpp-eks.yaml
```

Comment:
Could you introduce a variable for this version number (say `{{site.vppbranch}}`)?  Feel like that'd make it a lot easier to keep up to date in future.